golden_test/home_test.dart

```diff
@@ -0,0 +1,80 @@
+// Copyright 2019 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import 'package:gallery/main.dart';
+
+import 'testing/precache_images.dart';
+import 'testing/util.dart';
+
+const homeAssets = [
+  'assets/icons/cupertino/cupertino.png',
+  'assets/icons/material/material.png',
+  'assets/icons/reference/reference.png',
+  'assets/logo/flutter_logo.png',
+  'assets/studies/crane_card.png',
+  'assets/studies/crane_card_dark.png',
+  'assets/studies/rally_card.png',
+  'assets/studies/rally_card_dark.png',
+  'assets/studies/shrine_card.png',
+  'assets/studies/shrine_card_dark.png',
+  'assets/studies/fortnightly_card.png',
+  'assets/studies/fortnightly_card_dark.png',
+];
+
+void main() {
+  group('mobile', () {
+    testWidgets('home page matches golden screenshot', (tester) async {
+      await setUpBinding(tester);
```

Comment:
you might want to undo any bindings you set at the end of each test, to avoid any accidental issues. You can do that in a tearDown method instead of inside each test

Reply:
Added a tear down now to clear the test values:

`addTearDown(tester.binding.window.clearAllTestValues);`